### PR TITLE
Check in 133.py.

### PR DESCRIPTION
**Pull request by bunnygod**

### CODE REVIEW

File: Graph/Basic Graph Traversal/133.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        self.neighbors = neighbors if neighbors is not None else []
+"""
+
+# BFS Solution
+class Solution:
+    def cloneGraph(self, node: 'Node') -> 'Node':
+        if not node:
+            return node
+
+        graph = {}
```
Comment (mujingw) [inline Graph/Basic Graph Traversal/133.py:15]: The name `graph` is misleading here. It's not a representation of any graph; it's a mapping between those original vertices and their corresponding cloned ones.

File: Graph/Basic Graph Traversal/133.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        queue = deque([node])
+        graph[node] = Node(node.val, [])
+
+        while queue:
+            n = queue.popleft()
+
+            for neighbor in n.neighbors:
+                if neighbor not in graph:
```
Comment (mujingw) [inline Graph/Basic Graph Traversal/133.py:23]: Do you recognize the **_role_** this `graph` variable plays here (other than storing the cloned vertices)?

Comment (bunnygod) [reply Graph/Basic Graph Traversal/133.py:23]: It works as `seen`.

Comment (mujingw) [reply Graph/Basic Graph Traversal/133.py]: Great! This is exactly what I wanted to hear.

**Pull request merged**